jupyterhub/values.yaml

```diff
@@ -5,7 +5,7 @@ hub:
   cookieSecret:
   publicURL:
   uid: 1000
-  fsGid: 1000
+  fsGid: 100
```

Comment:
this isn't the right gid for the hub

Reply:
Ah, finally starting to grasp this better! I'll fix this!